<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PredictWindow</class>
 <widget class="QWidget" name="PredictWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>100</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>100</number>
   </property>
   <property name="bottomMargin">
    <number>100</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="label_header">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:dimgray;</string>
       </property>
       <property name="text">
        <string>پیشبینی قیمت خانه</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_predict_price">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:dimgray;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>10</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>20</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_zone_value">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color:dimgray;</string>
              </property>
              <property name="text">
               <string>درحال دریافت اطلاعات</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <property name="indent">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_meter_value">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color:dimgray;</string>
              </property>
              <property name="text">
               <string>درحال دریافت اطلاعات</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <property name="indent">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_bedroom_value">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color:dimgray;</string>
              </property>
              <property name="text">
               <string>درحال دریافت اطلاعات</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <property name="indent">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_year_value">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color:dimgray;</string>
              </property>
              <property name="text">
               <string>درحال دریافت اطلاعات</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <property name="indent">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_predict_price_value">
              <property name="font">
               <font>
                <pointsize>15</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color:teal;</string>
              </property>
              <property name="text">
               <string>درحال دریافت اطلاعات</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <property name="indent">
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>20</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_zone">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color:dimgray;</string>
              </property>
              <property name="text">
               <string>منطقه :</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <property name="indent">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_meter">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color:dimgray;</string>
              </property>
              <property name="text">
               <string>متراژ :</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <property name="indent">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_bedroom">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color:dimgray;</string>
              </property>
              <property name="text">
               <string>تعداد اتاق :</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <property name="indent">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_year">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color:dimgray;</string>
              </property>
              <property name="text">
               <string>سال ساخت :</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <property name="indent">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_predict_price">
              <property name="font">
               <font>
                <pointsize>15</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color:dimgray;</string>
              </property>
              <property name="text">
               <string>قیمت پیشبینی شده خانه مورد نظر :</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <property name="indent">
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_network_information">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:dimgray;</string>
       </property>
       <property name="text">
        <string>اطلاعات شبکه یادگیری</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_network_information">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>10</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>20</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_train_data_number">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color:dimgray;</string>
              </property>
              <property name="text">
               <string>Number of train data : </string>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <property name="indent">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_test_data_number">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color:dimgray;</string>
              </property>
              <property name="text">
               <string>Number of test data : </string>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <property name="indent">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_mse">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color:dimgray;</string>
              </property>
              <property name="text">
               <string>Mean squared error :</string>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <property name="indent">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_rmse">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color:dimgray;</string>
              </property>
              <property name="text">
               <string>Root mean squared error :</string>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <property name="indent">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_mae">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color:dimgray;</string>
              </property>
              <property name="text">
               <string>Mean absolute error :</string>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <property name="indent">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_mpd">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color:dimgray;</string>
              </property>
              <property name="text">
               <string>Mean poisson deviance :</string>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <property name="indent">
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>20</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_train_data_number_value">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color:dimgray;</string>
              </property>
              <property name="text">
               <string>processing</string>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <property name="indent">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_test_data_number_value">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color:dimgray;</string>
              </property>
              <property name="text">
               <string>processing</string>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <property name="indent">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_mse_value">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color:dimgray;</string>
              </property>
              <property name="text">
               <string>processing</string>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <property name="indent">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_rmse_value">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color:dimgray;</string>
              </property>
              <property name="text">
               <string>processing</string>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <property name="indent">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_mae_value">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color:dimgray;</string>
              </property>
              <property name="text">
               <string>processing</string>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <property name="indent">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_mpd_value">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color:dimgray;</string>
              </property>
              <property name="text">
               <string>processing</string>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <property name="indent">
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontal_layout_buttons">
       <property name="spacing">
        <number>30</number>
       </property>
       <property name="topMargin">
        <number>30</number>
       </property>
       <item>
        <widget class="QPushButton" name="button_search_house">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:teal;
background-color:lightgray;
</string>
         </property>
         <property name="text">
          <string>جست و جوی خانه</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_back">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:teal;
background-color:lightgray;
</string>
         </property>
         <property name="text">
          <string>بازگشت</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
